Library/Homebrew/utils.rb

```diff
@@ -376,6 +376,14 @@ def ignore_interrupts(opt = nil)
   trap("INT", std_trap)
 end
 
+def capture_stderr
```

Comment:
I don't think `shutup` returns the captured output. Would you be open to it being refactored to return the captured `stdout` and `stderr`?